app/views/brexit_checker/_results_criteria.html.erb

```diff
@@ -0,0 +1,21 @@
+<% if criteria.present? %>
+  <aside class="govuk-grid-column-one-third">
+    <div class="brexit-checker__criteria">
+      <% if local_assigns[:heading] %>
+        <h3 class="govuk-heading-m"><%= heading %></h3>
+      <% end %>
+      <p class="govuk-body govuk-!-font-size-16">
```

Comment:
Same comment about avoiding override classes - can we add a class which we can replace instances of `govuk-body govuk-!-font-size-16` with?

Reply:
humm yeah these are a bit of copy paste on my part... they're all over this app.
Could change them all at some point but will remove the ones I've brought in here.

Reply:
@huwd I think you've removed them, but not replaced them with anything - is that ok? It's now using the default 19px font-size, so text is quite a lot bigger.

Reply:
yeah, in the end I move these down to a more specific set of classes. I reckon the css on bullet points was overriding the css in the class I replaced it with on this div.